concepticondata/conceptlists/Johansson-2020-344.tsv

```diff
@@ -1,345 +1,346 @@
+Johansson-2020-344-227	227	THAT YONDER			0	that yonder
+Johansson-2020-344-228	228	THEN	2921	THEN	0	then
+Johansson-2020-344-229	229	THERE	1937	THERE	0	there
+Johansson-2020-344-230	230	THERE YONDER			0	there yonder
+Johansson-2020-344-231	231	THICK	1244	THICK	0	thick
+Johansson-2020-344-232	232	THIGH	471	UPPER LEG (THIGH)	0	leg (upper, thigh)
+Johansson-2020-344-233	233	THIN	2308	THIN	0	thin
+Johansson-2020-344-234	234	THINK	2271	THINK	1	think
+Johansson-2020-344-235	235	THIS	1214	THIS	1	this
+Johansson-2020-344-236	236	THREE	492	THREE	0	three
+Johansson-2020-344-237	237	THROAT	1346	THROAT	1	throat
+Johansson-2020-344-238	238	THUNDER	1150	THUNDER	0	thunder
+Johansson-2020-344-239	239	TIE	1917	TIE	1	tie
+Johansson-2020-344-240	240	TOE	1389	TOE	0	toe
+Johansson-2020-344-241	241	TONGUE	1205	TONGUE	1	tongue
+Johansson-2020-344-242	242	TOOTH	1380	TOOTH	0	tooth
+Johansson-2020-344-243	243	TOUCH	1892	TOUCH	0	touch
+Johansson-2020-344-244	244	TREE	906	TREE	0	tree
+Johansson-2020-344-245	245	TURN	1588	TURN	1	turn
+Johansson-2020-344-246	246	TWELVE	1707	TWELVE	0	twelve
+Johansson-2020-344-247	247	TWENTY	1710	TWENTY	0	twenty
+Johansson-2020-344-248	248	TWO	1498	TWO	0	two
+Johansson-2020-344-249	249	UGLY	1548	UGLY	0	ugly
+Johansson-2020-344-250	250	URINATE	592	PISS	0	urinate
+Johansson-2020-344-251	251	VOMIT	1278	VOMIT	0	vomit
+Johansson-2020-344-252	252	VULVA	1837	VULVA	0	vulva
+Johansson-2020-344-253	253	WANT	1784	WANT	0	want
+Johansson-2020-344-254	254	WATER	948	WATER	0	water
+Johansson-2020-344-255	255	WEAK	1601	WEAK	0	weak
+Johansson-2020-344-256	256	WET	1726	WET	0	wet
+Johansson-2020-344-257	257	WHAT?	1236	WHAT	1	what?
+Johansson-2020-344-258	258	WHEN?	1238	WHEN	0	when?
+Johansson-2020-344-259	259	WHERE?	1237	WHERE	1	where?
+Johansson-2020-344-260	260	WHITE	1335	WHITE	0	white
+Johansson-2020-344-261	261	WHO?	1235	WHO	1	who?
+Johansson-2020-344-262	262	WHY?	1270	WHY	0	why?
+Johansson-2020-344-263	263	WIDE	1243	WIDE	0	wide
+Johansson-2020-344-264	264	WING	1257	WING	0	wing
+Johansson-2020-344-265	265	WIND	960	WIND	0	wind
+Johansson-2020-344-266	266	WOMAN	962	WOMAN	0	woman
+Johansson-2020-344-267	267	WORD	1599	WORD	0	word
+Johansson-2020-344-268	268	WRONG	1390	WRONG	0	wrong
+Johansson-2020-344-269	269	YAWN	1028	YAWN	1	yawn
+Johansson-2020-344-270	270	YEAR	1226	YEAR	0	year
+Johansson-2020-344-271	271	YELLOW	1424	YELLOW	0	yellow
+Johansson-2020-344-272	272	YESTERDAY	1174	YESTERDAY	0	yesterday
+Johansson-2020-344-273	273	YOUNG	1207	YOUNG	0	young
+Johansson-2020-344-274	274	1SG	1209	I	1	1SG
+Johansson-2020-344-275	275	1PLI	1131	WE (INCLUSIVE)	1	1PLI
+Johansson-2020-344-276	276	1PLE	1130	WE (EXCLUSIVE)	1	1PLE
+Johansson-2020-344-277	277	2SG	1215	THOU	1	2SG
+Johansson-2020-344-278	278	2PL	1213	YOU	1	2PL
+Johansson-2020-344-279	279	3SG	262	HE OR SHE OR IT	1	3SG
+Johansson-2020-344-280	280	3PL	817	THEY	0	3PL
+Johansson-2020-344-281	281	mother's mother (female speaking)			0	MM (fs)
+Johansson-2020-344-281	281	mother's mother (female speaking)			0	MM (fs)
```

Comment:
That's strange, somehow there was a line added here.

Reply:
Thanks, I'll remove and check for other duplicates